### ISSUE TITLE
Translation: <language>

### ISSUE THREAD

**Issue by kjdhgfs5989709** [Rookiestyle/KeePassOTP#77]
[NOTE]: # (Please do not forget to attach the translation file)
Language: 
- [ ] New translation
- [ ] Update existing translation
- [ ] If updated: Only added missing Texts
- [ ] All relevant texts translated

[NOTE]: # (A text is relevant if the standard text in english language is NOT sufficient)

Hi, I do that for the first time here, dono if it is the right place. I got problems with keepass plugin language files. All language files are placed in:
C:\Program Files (x86)\KeePass Password Safe 2\Plugins\Translations
for example: KeePassOTP.de.language.xml and EarlyUpdateCheck.de.language.xml, actual versions
But everytime Keepass starts, I got the same error for every language file, which is placed there:
"KeePassOTP: Error parsing translation file. Fault in XML-Document (8,2) <html xmlns="> was not expected"
"Early Update Check: Error parsing translation file. Fault in XML-Document (8,2) <html xmlns="> was not expected"
Both plugins are (therefore) still in english, but not in german, as desired.
If I delete the files from the Translations-folder, no error. If I place the files somewhere else in the program folder, no error but also no translation. I really don't have any idea why. :( 
It isn't a problem with the german translation files, because the french one gives the same error. I do not have xml-skills and there are no other google hints for that, so I post it here in the hope, that someone can help.

**Comment by Rookiestyle** [comment]
Most likely you downloaded the translation files in an improper way - which can happen easily on Github :)

**Option 1**
* Navigate to the translation file in your browser
* Click the `Raw` link to download the xml file as such and not the entire website

**Option 2**
As you seem to use EarlyUpdateCheck
* Open it's options in KeePass (`Tools -> Early Update Check...`)
* Click `Manually update translation` 
If KeePass' language is german, this will download the german translation files

![image](https://user-images.githubusercontent.com/36454520/141281046-85793454-a9c9-4afb-9711-20ed24df7285.png)

**Comment by kjdhgfs5989709** [comment]
Great! Tried option 2, works!

> Most likely you downloaded the translation files in an improper way - which can happen easily on Github :)

Before I tried Firefox --> righklick on that xml-file-link --> Save Link As...

Good to know, that this isn't working ;)
I comared both versions, the one from the Early Update Check und die manually downloaded one with Firefox. It is completly crap! omg

![image](https://user-images.githubusercontent.com/94072809/141342299-6e527b4f-1eb0-4fd4-8d5c-619d48775faf.png)

Thanks for your help and thanks for that Plugin!!!
